TESTING.md

```diff
@@ -63,6 +63,12 @@ To run tests requiring a database, start the test databases using Docker, see [D
 
 ## Jest tips
 
+1. Update your dependencies:
+
+   ```sh
+   pnpm i
+   ```
+
```

Comment:
This doesn't belong under `Jest tips` I think

Maybe we move it to the top under a new subtitle like `Install dependencies`?

Reply:
I think someone looking to run unit tests will arrive at this section for specific help. It's kind of a precondition to update your dependencies to have your tests under a stable fixture. If this was in a different place, the reader will have to know where and have read the whole documentation. Having this step in here allows for the section to be self-sufficient.

Reply:
Having said that, feel free to go ahead and change it to a better place if you have a stronger opinion. I defer it to you.